Laud_Misc_610/CGH/_1.trig

```diff
@@ -28,8 +28,12 @@
     owl:sameAs <http://example.com/LL/lagin.trig#MiledEspane>;
     owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.trig#Mílid>.
 
+<< <#Militis>
+        rdfs:comment "This text is in Latin.  The predicate used ties the name to Old Irish specifically.  Maybe we should remove the specific tie to Old Irish and let it be any nominative" >>
+        prov:wasAttributedTo <https://orcid.org/0000-0002-7241-3264> .
+
 <#Herimon>
-    a foaf:Person;
+    a irishTitle:Rí;
     irishRel:nomName "Herimon";
     rel:childOf <#Militis>;
     owl:sameAs <http://example.com/LL/lagin.trig#h-Erimoin>;
```

Comment:
It's not explicitly stated in this text, but if Érimón is king of the northern half of Ireland, Eber is presumably king of the southern half. So should Eber be `a irishTitle:Rí` too?